### ISSUE TITLE
Migrate from Bullseye to Bookworm?

### ISSUE THREAD

**Issue by bmoscon** [Azure/azure-functions-docker#1045]
Are there plans to migrate from bullseye to bookworm? Specifically for the python azure functions docker image? Bullseye is a bit outdated and contains several vulnerable packages.

**Comment by snesm** [comment]
Agreed.  Debian 11 (bullseye) security [support](https://endoflife.date/debian) ends 01 July 2024.

**Comment by vivekjilla** [comment]
Thanks for raising this. The Python images have since moved to Debian Bookworm, and the latest images are on Ubuntu Noble. Since that migration is complete, we're closing this. If you spot a specific tag still on Bullseye, please open a new issue with the image/digest.
